gp-templates/translations.php

```diff
@@ -51,8 +56,8 @@
 		);
 		?>
 	</h2>
-	<?php gp_link_set_edit( $translation_set, $project, _x( '(edit)', 'translation set', 'glotpress' ) ); ?>
-	<?php gp_link_set_delete( $translation_set, $project, _x( '(delete)', 'translation set', 'glotpress' ) ); ?>
+	<?php gp_link_set_edit( $translation_set, $project, null, array( 'class' => 'button is-small' ) ); ?>
```

Comment:
Could you double check why this `_x()` part can be removed?

Reply:
The [`gp_link_set_edit()`](https://github.com/GlotPress/GlotPress/blob/7e0f20bbf19fd4e7c76ef96d29a829cf31b39f78/gp-includes/template-links.php#L247) function echoes the [`gp_link_set_edit_get()`](https://github.com/GlotPress/GlotPress/blob/7e0f20bbf19fd4e7c76ef96d29a829cf31b39f78/gp-includes/template-links.php#L226) function. Its default value is `__( 'Edit', 'glotpress' )`. 

The same happens with `Delete`, changing the functions' signatures and the default value.